scripts/add-validator/README.md

```diff
@@ -0,0 +1,38 @@
+# Add Validator
+
+This script will help us in adding the validator node to blockchain programmaticaly. 
+
+<br />
+
+## Requirements
+[NodeJs (v14.17.0 lts)](https://nodejs.org/en/download/ "NodeJs (v14.17.0 lts)")
+
+<br />
+
+## Configure app
+Open `.env` and update the with appropriate values.
```

Comment:
`and update it with the appropriate values.`

Reply:
updated.